Jenkinsfile

```
@@ -1,10 +1,15 @@
 pipeline {
     agent any
     stages {
+        stage('Identification') {
+            steps {
+                sh 'echo Change request author: ${CHANGE_AUTHOR} '
+            }
+        }
         stage('Confirmation') {
             when {
                 beforeInput true
-                changeRequest author: 'dependabot-preview'
+                changeRequest author: '[\\w]*dependabot[\\w]*', comparator: "REGEXP"
```

Did you verify that this is not enough?

```suggestion
                changeRequest author: 'dependabot', comparator: "REGEXP"
```

---

BTW, `-` is a word character (`\w`)

---

Perhaps this suffice I'm not so sure. I guess I can test this approach with my user to be sure.